src/mixins/filing-template-mixin.ts

```diff
@@ -721,12 +790,14 @@ export default class FilingTemplateMixin extends DateMixin {
     // store current Business Contact
     this.setBusinessContact({ ...entitySnapshot.authInfo.contact })
 
-    // store People And Roles
-    let orgPersons = filing.correction.parties || entitySnapshot.orgPersons
-    // exclude Completing Party
-    // (it is managed separately and added to the filing in buildCorrectionFiling())
-    orgPersons = orgPersons.filter(op => !(op?.roles.some(role => role.roleType === RoleTypes.COMPLETING_PARTY)))
-    this.setPeopleAndRoles(cloneDeep(orgPersons))
+    if (!this.isCoopCorrectionFiling) {
+      // store People And Roles
+      let orgPersons = filing.correction.parties || entitySnapshot.orgPersons
+      // exclude Completing Party
+      // (it is managed separately and added to the filing in buildCorrectionFiling())
+      orgPersons = orgPersons.filter(op => !(op?.roles.some(role => role.roleType === RoleTypes.COMPLETING_PARTY)))
```

Comment:
Ok, I didn't change this code, just put in a if condition above it to exclude it from Coop corrections

Reply:
Yeah I know.

I noticed other filings in here also filter right in the main method instead of using a helper.

We should fix this later. OK for now. Resolving.